src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(MAX_NUMBER + 1));
```

Comment:
You are creating a new instance of `ColorSupplier` every time you call `getRandomBall()`. Consider creating a single instance of `ColorSupplier` as a class field and reusing it.